bin/fetch.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import os
+import sys
+
+PLATFORM = os.name
+if PLATFORM == 'nt':
+    import zipfile
+
+    def extractall(archive):
+        with zipfile.ZipFile(archive, 'r') as zip:
+            zip.extractall()
```

Comment:
You should probably change `zip` to something else to avoid future confusion with the built-in `zip` function.